### PR TITLE
Maps implementation in Java

### PR DESCRIPTION
**Pull request by temucher**
- Rewrote the maps implementation into java
- Removed Maps.java and maps.xml and replaced them with MapsActivity.java and activity_maps.xml

@justinnhli   @crzeller11

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```
Comment (justinnhli) [inline app/src/main/AndroidManifest.xml]: Isn't this a duplicate of the line before and therefore unnecessary?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -36,11 +38,14 @@
...
-        <activity android:name=".Nutrition"/>
+        <activity android:name=".Exercise" />
+        <activity android:name=".Nutrition" />
         <activity
-            android:name=".NutritionDetailActivity">
+            android:name=".MapsActivity"
+            android:label="@string/title_activity_maps">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value="practicum.getfitla_v3.MainActivity" />
```
Comment (justinnhli) [inline app/src/main/AndroidManifest.xml:47]: I am confused by these changes here. Why does this remove NutritionDetailActivity?

File: app/src/main/java/practicum/getfitla_v3/MainActivity.java [hunk 3]
```diff
@@ -1,13 +1,10 @@
 package practicum.getfitla_v3;
 
 import android.content.Intent;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.support.v7.widget.RecyclerView;
+import android.support.v7.app.AppCompatActivity;
```
Comment (justinnhli) [inline app/src/main/java/practicum/getfitla_v3/MainActivity.java:5]: Ditto. Why does this remove RecyclerView? Or is it an unused import?

File: app/src/main/java/practicum/getfitla_v3/MapsActivity.java [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+            if (mLocationPermissionGranted) {
+                Task locationResult = mFusedLocationProviderClient.getLastLocation();
+                locationResult.addOnCompleteListener(this, new OnCompleteListener() {
+                    @Override
+                    public void onComplete(@NonNull Task task) {
+                        if (task.isSuccessful()) {
+                            lastLocation = (Location) task.getResult();
+                            System.out.println("Test");
+                            mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(
+                                    // FIXME: .getLatitude() method throws null pointer exception
```
Comment (crzeller11) [inline app/src/main/java/practicum/getfitla_v3/MapsActivity.java]: If this is no longer an issue, remove it

**Review by crzeller11 — CHANGES_REQUESTED** [review]
I don't have many comments, other than a few of the redundancies that Justin pointed out. Most of this code is taken directly from the documentation, so it's quite clean.

**Comment by temucher** [comment]
Tried fixing things and pushing again, as per @blackjamx5 

@justinnhli  @crzeller11

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (temucher) [reply app/src/main/AndroidManifest.xml:47]: Added NutritionDetailActivity back in

File: app/src/main/java/practicum/getfitla_v3/MainActivity.java (hunk 3 above)
Comment (temucher) [reply app/src/main/java/practicum/getfitla_v3/MainActivity.java:5]: Yes, it's unused.

**Review by crzeller11 — APPROVED** [review]
Looks good

**Pull request merged**